website/docs/feature-flag-tutorials/react/examples.md

```diff
@@ -0,0 +1,627 @@
+
+
+![Gradual Rollout configure strategy constraint image](/img/react-ex-grad-rollout-edit.png)
+
+
+This will take you to the gradual rollout form. Click on the ‘Add constraint’ button.
+
+
+![Add constraint button image](/img/react-ex-add-constraint-btn.png)
+
+
+Let’s say we are experimenting with releasing the notifications feature for a limited time. We want to release it to **all users**, capture some usage data to compare it to the old experience, and then automatically turn it off.
+
+We can configure the constraint in the form to match these requirements:
+
+
+![Create constraint image](/img/react-ex-constraint-form.png)
+
+
+* The context field is set to **currentTime**
+* The operator is set to **DATE_BEFORE**
+* The date time is set (to any time) in the future
+
+Once you’ve filled out the proper constraint fields, click ‘Done’ and save the strategy.
+
+
+![Graduall rollout with constraint image](/img/react-ex-rollout-with-constraint.png)
+
+
+Your release process is now configured with a datetime-based strategy constraint.
+
+Alternatively, you can send an API command to apply the same requirements:
+
+```
+curl --location --request PUT 'http://localhost:4242/api/admin/projects/default/features/newNotificationsBadge/environments/development/strategies/806ebcbd-bb03-4713-8081-7dca3905e612' \
+    --header 'Authorization: INSERT_API_KEY' \
+    --header 'Content-Type: application/json' \
+    --data-raw '{
+  "name": "flexibleRollout",
+  "title": "",
+  "constraints": [
+    {
+      "value": "2024-02-27T17:00:00.000Z",
+      "values": [],
+      "inverted": false,
+      "operator": "DATE_BEFORE",
+      "contextName": "currentTime",
+      "caseInsensitive": false
+    }
+  ],
+  "parameters": {
+    "rollout": "50",
+    "stickiness": "default",
+    "groupId": "newNotificationsBadge"
+  },
+  "variants": [],
+  "segments": [],
+  "disabled": false
+}'
+```
+
+Read our documentation for more context on the robustness of [strategy constraint configurations](/reference/strategy-constraints) and use cases.
+
+
+## A/B Testing in React
+
+
+A/B testing is a common way for teams to test out how users interact with two or more versions of a new feature that is released. At Unleash, we call these [variants](/reference/feature-toggle-variants).
+
+We can expose a particular version of the feature to select user bases when a flag is enabled. From there, a way to use the variants is to view the performance metrics and see which is more efficient.
+
+We can create several variations of this feature to release to users and gather performance metrics to determine which one yields better results. While teams may have different goals for measuring performance, Unleash enables you to configure strategy for the feature variants within your application/service and the platform.
```

Comment:
Should "configure strategy" be "configure *a* strategy"? "configure strategies"? Something else?